### ISSUE TITLE
[accessibility]: accessibility issue with dotshell

### ISSUE THREAD

**Issue by cedricpelle** [carbon-design-system/carbon-for-ibm-dotcom#9036]
### Description

I am using the dotshell example to start
and i have some violations when using the IBM Equal Access Accessibility Checker
[Accessibility_Report-IBM Terms(10).xlsx](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/files/8976875/Accessibility_Report-IBM.Terms.10.xlsx)
r

![image](https://user-images.githubusercontent.com/38075847/175556505-4cfc07e7-a9d1-4fa3-bd91-d5583337fb8a.png)

### Component(s) impacted

https://wwwstage.ibm.com/support/customer/csol/terms/carbon/index-dotshell.html

### Browser

Chrome, Safari, Firefox, Microsoft Edge

### Carbon for IBM.com version

v1.35.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

https://wwwstage.ibm.com/support/customer/csol/terms/carbon/index-dotshell.html

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://wwwstage.ibm.com/support/customer/csol/terms/carbon/index-dotshell.html

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
